concent_api/concent_golem_messages/tests/factories.py

```diff
@@ -0,0 +1,69 @@
+import random
+
+from ethereum.utils import denoms
+from golem_messages.factories.helpers import MessageFactory
+import factory.fuzzy
+
+from concent_golem_messages.message import NonceAbstractMessage
+from concent_golem_messages.message import SignedTransaction
+from concent_golem_messages.message import TransactionRejected
+from concent_golem_messages.message import TransactionSigningRequest
+
+
+class NonceAbstractMessageFactory(MessageFactory):
+    class Meta:
+        model = NonceAbstractMessage
+
+    nonce = factory.LazyFunction(
+        lambda: random.randint(0, denoms.turing)
+    )
```

Comment:
@kbeker https://github.com/golemfactory/concent/pull/687#discussion_r207898049
> Please add tests for messages which will have nonce bigger than it is possible and check if it will raise exception

:+1: done, but tests are in `test_message.py`.